components/tooltip/tooltip-help.js

```diff
@@ -76,6 +77,9 @@ class TooltipHelp extends FocusMixin(FocusVisiblePolyfillMixin(LitElement)) {
 			d2l-tooltip {
 				cursor: text;
 			}
+			:host([skeleton]) button.d2l-skeletize {
```

Comment:
If we use `:host([skeleton]) #d2l-tooltip-help-text.d2l-skeletize` selector, we can avoid the `!important`. We try to only rely on `!important` when absolutely necessary.